js/src/dom/data.js

```
@@ -21,7 +21,7 @@ export default {
 
     // make it clear we only want one instance per element
     // can be removed later when multiple key/instances are fine to be used
-    if (!instanceMap.has(key) && instanceMap.size !== 0) {
```

In case of same key, could just silently return, without overriding it